tools/pytest.ini

```
@@ -28,3 +28,7 @@ filterwarnings =
     always:the imp module is deprecated in favour of importlib:DeprecationWarning
     # https://github.com/web-platform-tests/wpt/issues/39827
     always:pkg_resources is deprecated as an API:DeprecationWarning
+    # mozdevice uses pipes module
+    ignore:'pipes' is deprecated and slated for removal in Python 3.13:DeprecationWarning
+    # mozrunner uses telnetlib module
+    ignore:'telnetlib' is deprecated and slated for removal in Python 3.13:DeprecationWarning
```

```suggestion
    # mozdevice uses pipes module
    ignore:'pipes' is deprecated and slated for removal in Python 3:DeprecationWarning
    # mozrunner uses telnetlib module
    ignore:'telnetlib' is deprecated and slated for removal in Python 3:DeprecationWarning
```

Let's avoid being overly specific, given deprecations are often pushed back a release or two.

But also move this up to all the ignores above, leaving the things we care about at the bottom?